src/c_kzg_4844.c

```diff
@@ -1388,29 +1385,26 @@ static C_KZG_RET verify_kzg_proof_batch(
 
     /* Compute the random lincomb challenges */
     ret = compute_r_powers(
-        r_powers, commitments_g1, evaluation_challenges_fr, ys_fr, proofs_g1, n
+        r_powers, commitments_g1, zs_fr, ys_fr, proofs_g1, n
     );
     if (ret != C_KZG_OK) goto out;
 
     /* Compute \sum r^i * Proof_i */
     g1_lincomb_naive(&proof_lincomb, proofs_g1, r_powers, n);
 
     for (size_t i = 0; i < n; i++) {
-        g1_t ys_encrypted;
```

Comment:
Why not have this variable in reduced scope? I think it's better to have it in here to signify to the reader that it's not used anywhere else.